app/controllers/restaurant_notes_controller.rb

```diff
@@ -0,0 +1,18 @@
+class RestaurantNotesController < ApplicationController
+  def create
+    restaurant = Restaurant.find(params[:restaurant_id])
+    @restaurant_note = restaurant.restaurant_notes.new(restaurant_notes_params)
```

Comment:
@yukimurasawa ここも、build を使った書き方に変えられると良いですね 💪 
```suggestion
    @restaurant_note = restaurant.restaurant_notes.build(restaurant_notes_params)
```